pkg/sdk/urls_test.go

```diff
@@ -0,0 +1,76 @@
+package sdk_test
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/pokt-network/poktroll/pkg/sdk"
+)
+
+func TestHostToWebsocketURL(t *testing.T) {
+	tests := []struct {
+		name    string
+		hostUrl string
+		want    string
+	}{
+		{
+			name:    "Test HTTPS",
+			hostUrl: "https://poktroll.com",
+			want:    "wss://poktroll.com/websocket",
+		},
+		{
+			name:    "Test HTTP",
+			hostUrl: "http://poktroll.com",
+			want:    "ws://poktroll.com/websocket",
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			u, _ := url.Parse(tc.hostUrl)
+			got := sdk.HostToWebsocketURL(u)
+			if got != tc.want {
+				t.Fatalf("expected %s; got %s", tc.want, got)
+			}
```

Comment:
We use the [testify](https://github.com/stretchr/testify) testing library, so you should be able to do:
```suggestion
			require.Equal(t, got, tc.want)
```